Issue4~6の修正

coffta opened the pull request:
Issue4〜6の解消

coffta committed:
Issue4

coffta committed:
Revert "Issue4"

This reverts commit 42f2d55aa2fc5171636009d8ee79f3eba397250d.

coffta committed:
Issue4を再度やり直し

coffta committed:
Issue5

coffta committed:
Issue6

ShotaOtake commented:
レビュー依頼を頂きありがとうございます！

内容を拝見したところ、こちらのプルリクエストにて、**コンフリクト**が起きているようです。
まずはこちらを解消頂けますと幸いです。

githubのブラウザエディタにてコンフリクトを解消できますので、**「Resolve conflicts」**をクリックして解消していきましょう！

完了次第、改めてレビュー依頼フォームにてご提出をお願い致します！

coffta committed:
Merge branch 'main' into second_training

ShotaOtake commented:
レビュー依頼を頂きありがとうございます！
確認させて頂いたところ、下記の添付がなされていないようにお見受けしました。

```
予定を入力し「保存」ボタンを押すとデータが保存され、日付のボックスに表示されるキャプチャ動画
```

お手数ではございますが、改めて**キャプチャ動画**を添付の上、再度レビュー依頼を頂けますと幸いです。宜しくお願い致します！

coffta commented:
キャプチャ動画のURLはこちらです。
https://gyazo.com/d5f5d78aeb3ec373f4026d33a073a169
ご確認のほどよろしくお願いいたします。

sh10n7 commented on app/controllers/calendars_controller.rb:
```diff
@@ -15,7 +15,7 @@ def create
 
   def plan_params
-    params.require(:calendars).permit(:date, :plan)
+    params.require(:plan).permit(:date, :plan, :calendars)
```
```
params.require(:plan).permit(:date, :plan, :calendars)
```
上記のストロングパラメーターの記載を再度見直してみましょう！
permitの値に入力するのは、「保存したい値のキー」となります。

データベースに保存したい値を確認し、再度すトロングパラメーター内の記述を修正しましょう。

sh10n7 commented on app/views/calendars/index.html.erb:
```diff
@@ -1,24 +1,28 @@
-        <% end %>
-      </ul>
-    </div>
+<div>
```
issue3の不要なdivタグが戻ってしまっているようです。
再度不要なタグがないかご確認いただき、修正をお願い致します！

mihot117 replied:
issue3の不要なdivタグがまだ存在しています！
再度の修正をお願いいたします。

sh10n7 reviewed (commented):
レビュー依頼ありがとうございます。
修正点確認させていただきました。

追加で修正いただきたい箇所をコメントさせていただきましたので、ご確認をお願いいたします。

また、修正後はコミットプッシュを確認いただき、再度**レビュー依頼フォーム**のご提出
をお願いいたします。

ご不明点や疑問点ございましたら、チャット・コネクトメンターにご相談ください！
よろしくお願いいたします。

coffta committed:
ストロングパラメーター内記述と不要なdivの修正

キャプチャ動画のURLはこちらです。ご確認よろしくお願いします。https://gyazo.com/573bc95e875ff1ef517681f2f420c0c5

coffta committed:
Merge branch 'second_training' of https://github.com/coffta/training_curriculum_rails7 into second_training

mihot117 reviewed (commented):
レビュー依頼ありがとうございます。

修正点確認させていただきました。
追加でコメントいたしましたので、ご確認いただけますと幸いです。

修正後、再度レビュー依頼フォームをご提出ください。

ご対応のほど、よろしくお願いいたします。

coffta committed:
不要なdivタグの削除修正

mihot117 reviewed (commented):
レビュー依頼ありがとうございます！

今回は**LGTM**とさせていただきます。

これでissue1~6全てのissueが解消されました。お疲れ様でした！
忘れずにこのブランチをMergeしていただき、ローカルのMasterにPullをお願いいたします。

次のカリキュラムも頑張っていきましょう。
引き続き、応援しております！
